tools/oeutil/host/generate_evidence.cpp

```diff
@@ -19,7 +22,13 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include "oegenerate_u.h"
+#if defined(__linux__)
+#include <unistd.h>
+#elif defined(_WIN32)
+#define strcasecmp stricmp
+#include <windows.h>
+#endif
```

Comment:
Not required in this PR but it would be good to put these sort of thing into a .h file that can be included from multiple files rather that putting ifdefs into every .cpp file